src/service/visitsService.js

```
@@ -0,0 +1,52 @@
+    }
+  }
+
+  static async getHomeVisitDatesByDateForParent(date) {
+    try {
+      const response = await axiosInstance.get(`/visits/list/parent/${date}`);
+      return response.data;
+    } catch (error) {
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)